src/main/java/com/example/viacademy/entities/OrderItem.java

```diff
@@ -0,0 +1,36 @@
+package com.example.viacademy.entities;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "orderItems")
+@Getter
+@Setter
+public class OrderItem {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Integer quantity;
+
+    @ManyToOne
+    @JoinColumn(name = "course_id")
+    @JsonManagedReference
+    private Course course;
```

Comment:
in this case when the user use this table, the course can be used and the consultancy as null or inverse. we can avoid that making an abstract class and course and consultancy will inherit that abstract class, and as a result in this table will be necessary only one column.